root@masterhost:~/ansible_modules_redirection_test4/ansible-junos-stdlib/ansible_collections# ANSIBLE_DEBUG=1 ansible-playbook -i inventory junos_facts.yml 
537030 1742792805.68419: starting run
537030 1742792805.68890: Added group all to inventory
537030 1742792805.68898: Added group ungrouped to inventory
537030 1742792805.68907: Group all now contains ungrouped
537030 1742792805.68914: Examining possible inventory source: /root/ansible_modules_redirection_test4/ansible-junos-stdlib/ansible_collections/inventory
537030 1742792805.81623: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/cache
537030 1742792805.81692: Loading CacheModule 'memory' from /usr/local/lib/python3.12/site-packages/ansible/plugins/cache/memory.py
537030 1742792805.81720: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/inventory
537030 1742792805.81782: Loading InventoryModule 'host_list' from /usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/host_list.py
537030 1742792805.81851: Loaded config def from plugin (inventory/script)
537030 1742792805.81857: Loading InventoryModule 'script' from /usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/script.py
537030 1742792805.81897: Loading InventoryModule 'auto' from /usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/auto.py
537030 1742792805.81977: Loaded config def from plugin (inventory/yaml)
537030 1742792805.81983: Loading InventoryModule 'yaml' from /usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/yaml.py
537030 1742792805.82071: Loading InventoryModule 'ini' from /usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/ini.py
537030 1742792805.82453: Loading InventoryModule 'toml' from /usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/toml.py
537030 1742792805.82460: Attempting to use plugin host_list (/usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/host_list.py)
537030 1742792805.82465: Attempting to use plugin script (/usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/script.py)
537030 1742792805.82475: Attempting to use plugin auto (/usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/auto.py)
537030 1742792805.82478: Attempting to use plugin yaml (/usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/yaml.py)
537030 1742792805.82510: Loading data from /root/ansible_modules_redirection_test4/ansible-junos-stdlib/ansible_collections/inventory
537030 1742792805.82571: /root/ansible_modules_redirection_test4/ansible-junos-stdlib/ansible_collections/inventory was not parsable by yaml
537030 1742792805.82641: Attempting to use plugin ini (/usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/ini.py)
537030 1742792805.82707: Added group junos to inventory
537030 1742792805.82721: set inventory_file for 10.220.10.70
537030 1742792805.82727: set inventory_dir for 10.220.10.70
537030 1742792805.82730: Added host 10.220.10.70 to inventory
537030 1742792805.82735: Added host 10.220.10.70 to group junos
537030 1742792805.82745: set ansible_network_os for junos
537030 1742792805.82754: set ansible_ssh_user for junos
537030 1742792805.82759: set ansible_ssh_pass for junos
537030 1742792805.82765: set ansible_connection for junos
537030 1742792805.82770: set ansible_network_cli_ssh_type for junos
537030 1742792805.82775: Reconcile groups and hosts in inventory.
537030 1742792805.82781: Group all now contains junos
537030 1742792805.82886: '/usr/local/lib/python3.12/site-packages/ansible/plugins/vars/__init__' skipped due to reserved name
537030 1742792805.83012: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments
537030 1742792805.83069: Loading ModuleDocFragment 'vars_plugin_staging' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/vars_plugin_staging.py
537030 1742792805.83095: Loaded config def from plugin (vars/host_group_vars)
537030 1742792805.83099: Loading VarsModule 'host_group_vars' from /usr/local/lib/python3.12/site-packages/ansible/plugins/vars/host_group_vars.py (found_in_cache=False, class_only=True)
537030 1742792805.83109: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/vars
537030 1742792805.83119: Loading VarsModule 'host_group_vars' from /usr/local/lib/python3.12/site-packages/ansible/plugins/vars/host_group_vars.py (found_in_cache=True, class_only=False)
537030 1742792805.83162: Loading CacheModule 'memory' from /usr/local/lib/python3.12/site-packages/ansible/plugins/cache/memory.py (found_in_cache=True, class_only=False)
537030 1742792805.83495: '/usr/local/lib/python3.12/site-packages/ansible/plugins/connection/__init__' skipped due to reserved name
537030 1742792805.83586: Loading ModuleDocFragment 'connection_pipelining' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/connection_pipelining.py
537030 1742792805.83624: Loaded config def from plugin (connection/local)
537030 1742792805.83632: Loading Connection 'local' from /usr/local/lib/python3.12/site-packages/ansible/plugins/connection/local.py (found_in_cache=False, class_only=True)
537030 1742792805.83959: Loaded config def from plugin (connection/paramiko_ssh)
537030 1742792805.83966: Loading Connection 'paramiko_ssh' from /usr/local/lib/python3.12/site-packages/ansible/plugins/connection/paramiko_ssh.py (found_in_cache=False, class_only=True)
537030 1742792805.84776: Loading ModuleDocFragment 'connection_pipelining' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/connection_pipelining.py (found_in_cache=True, class_only=False)
537030 1742792805.84814: Loaded config def from plugin (connection/psrp)
537030 1742792805.84820: Loading Connection 'psrp' from /usr/local/lib/python3.12/site-packages/ansible/plugins/connection/psrp.py (found_in_cache=False, class_only=True)
537030 1742792805.85318: Loading ModuleDocFragment 'connection_pipelining' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/connection_pipelining.py (found_in_cache=True, class_only=False)
537030 1742792805.85357: Loaded config def from plugin (connection/ssh)
537030 1742792805.85363: Loading Connection 'ssh' from /usr/local/lib/python3.12/site-packages/ansible/plugins/connection/ssh.py (found_in_cache=False, class_only=True)
537030 1742792805.87013: Loading ModuleDocFragment 'connection_pipelining' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/connection_pipelining.py (found_in_cache=True, class_only=False)
537030 1742792805.87055: Loaded config def from plugin (connection/winrm)
537030 1742792805.87061: Loading Connection 'winrm' from /usr/local/lib/python3.12/site-packages/ansible/plugins/connection/winrm.py (found_in_cache=False, class_only=True)
537030 1742792805.87100: '/usr/local/lib/python3.12/site-packages/ansible/plugins/shell/__init__' skipped due to reserved name
537030 1742792805.87166: Loading ModuleDocFragment 'shell_windows' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/shell_windows.py
537030 1742792805.87217: Loaded config def from plugin (shell/cmd)
537030 1742792805.87222: Loading ShellModule 'cmd' from /usr/local/lib/python3.12/site-packages/ansible/plugins/shell/cmd.py (found_in_cache=False, class_only=True)
537030 1742792805.87243: Loading ModuleDocFragment 'shell_windows' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/shell_windows.py (found_in_cache=True, class_only=False)
537030 1742792805.87292: Loaded config def from plugin (shell/powershell)
537030 1742792805.87297: Loading ShellModule 'powershell' from /usr/local/lib/python3.12/site-packages/ansible/plugins/shell/powershell.py (found_in_cache=False, class_only=True)
537030 1742792805.87357: Loading ModuleDocFragment 'shell_common' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/shell_common.py
537030 1742792805.87488: Loaded config def from plugin (shell/sh)
537030 1742792805.87494: Loading ShellModule 'sh' from /usr/local/lib/python3.12/site-packages/ansible/plugins/shell/sh.py (found_in_cache=False, class_only=True)
537030 1742792805.87535: '/usr/local/lib/python3.12/site-packages/ansible/plugins/become/__init__' skipped due to reserved name
537030 1742792805.87642: Loaded config def from plugin (become/runas)
537030 1742792805.87648: Loading BecomeModule 'runas' from /usr/local/lib/python3.12/site-packages/ansible/plugins/become/runas.py (found_in_cache=False, class_only=True)
537030 1742792805.87814: Loaded config def from plugin (become/su)
537030 1742792805.87819: Loading BecomeModule 'su' from /usr/local/lib/python3.12/site-packages/ansible/plugins/become/su.py (found_in_cache=False, class_only=True)
537030 1742792805.87952: Loaded config def from plugin (become/sudo)
537030 1742792805.87957: Loading BecomeModule 'sudo' from /usr/local/lib/python3.12/site-packages/ansible/plugins/become/sudo.py (found_in_cache=False, class_only=True)
537030 1742792805.87994: Loading data from /root/ansible_modules_redirection_test4/ansible-junos-stdlib/ansible_collections/junos_facts.yml
537030 1742792805.88649: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/callback
537030 1742792805.88743: Loading ModuleDocFragment 'default_callback' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/default_callback.py
537030 1742792805.88888: Loading ModuleDocFragment 'result_format_callback' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/result_format_callback.py
537030 1742792805.88944: Loaded config def from plugin (callback/default)
537030 1742792805.88950: Loading CallbackModule 'default' from /usr/local/lib/python3.12/site-packages/ansible/plugins/callback/default.py
537030 1742792805.88995: '/usr/local/lib/python3.12/site-packages/ansible/plugins/callback/__init__' skipped due to reserved name
537030 1742792805.89000: Loading CallbackModule 'default' from /usr/local/lib/python3.12/site-packages/ansible/plugins/callback/default.py (found_in_cache=True, class_only=True)
537030 1742792805.89922: Loaded config def from plugin (callback/junit)
537030 1742792805.89928: Loading CallbackModule 'junit' from /usr/local/lib/python3.12/site-packages/ansible/plugins/callback/junit.py (found_in_cache=False, class_only=True)
537030 1742792805.89973: Loading ModuleDocFragment 'result_format_callback' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/result_format_callback.py (found_in_cache=True, class_only=False)
537030 1742792805.90026: Loaded config def from plugin (callback/minimal)
537030 1742792805.90033: Loading CallbackModule 'minimal' from /usr/local/lib/python3.12/site-packages/ansible/plugins/callback/minimal.py (found_in_cache=False, class_only=True)
537030 1742792805.90072: Loading CallbackModule 'oneline' from /usr/local/lib/python3.12/site-packages/ansible/plugins/callback/oneline.py (found_in_cache=False, class_only=True)
537030 1742792805.90127: Loaded config def from plugin (callback/tree)
537030 1742792805.90132: Loading CallbackModule 'tree' from /usr/local/lib/python3.12/site-packages/ansible/plugins/callback/tree.py (found_in_cache=False, class_only=True)
537030 1742792805.90146: in VariableManager get_vars()
537030 1742792805.90186: done with get_vars()
537030 1742792805.90197: in VariableManager get_vars()
537030 1742792805.90212: done with get_vars()
537030 1742792805.90218: variable 'omit' from source: magic vars
537030 1742792805.90243: in VariableManager get_vars()
537030 1742792805.90258: done with get_vars()
537030 1742792805.90277: variable 'omit' from source: magic vars

PLAY [Test Junos Facts Module] ******************************************************************************************************************************
537030 1742792805.95313: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/strategy
537030 1742792805.95429: Loading StrategyModule 'linear' from /usr/local/lib/python3.12/site-packages/ansible/plugins/strategy/linear.py
537030 1742792805.95519: getting the remaining hosts for this loop
537030 1742792805.95525: done getting the remaining hosts for this loop
537030 1742792805.95528: getting the next task for host 10.220.10.70
537030 1742792805.95537: done getting next task for host 10.220.10.70
537030 1742792805.95541:  ^ task is: TASK: Gather Junos Facts
537030 1742792805.95545:  ^ state is: HOST STATE: block=2, task=1, rescue=0, always=0, handlers=0, run_state=1, fail_state=0, pre_flushing_run_state=None, update_handlers=True, pending_setup=False, tasks child state? (None), rescue child state? (None), always child state? (None), did rescue? False, did start at task? False
537030 1742792805.95551: getting variables
537030 1742792805.95555: in VariableManager get_vars()
537030 1742792805.95572: Calling all_inventory to load vars for 10.220.10.70
537030 1742792805.95576: Calling groups_inventory to load vars for 10.220.10.70
537030 1742792805.95581: Calling all_plugins_inventory to load vars for 10.220.10.70
537030 1742792805.95598: Calling all_plugins_play to load vars for 10.220.10.70
537030 1742792805.95603: Calling groups_plugins_inventory to load vars for 10.220.10.70
537030 1742792805.95609: Calling groups_plugins_play to load vars for 10.220.10.70
537030 1742792805.95649: '/usr/local/lib/python3.12/site-packages/ansible/plugins/connection/__init__' skipped due to reserved name
537030 1742792805.95694: done with get_vars()
537030 1742792805.95705: done getting variables
537030 1742792805.96076: Loading ActionModule 'ansible_collections.juniper.device.plugins.action.facts' from /root/ansible_modules_redirection_test4/ansible-junos-stdlib/ansible_collections/juniper/device/plugins/action/facts.py (found_in_cache=False, class_only=True)

TASK [Gather Junos Facts] ***********************************************************************************************************************************
537030 1742792805.96099: entering _queue_task() for 10.220.10.70/junos_facts
537030 1742792805.96105: Creating lock for junos_facts
537030 1742792805.96403: worker is 1 (out of 1 available)
537030 1742792805.96452: exiting _queue_task() for 10.220.10.70/junos_facts
537030 1742792805.96476: done queuing things up, now waiting for results queue to drain
537030 1742792805.96480: waiting for pending results...
537030 1742792805.96803: running TaskExecutor() for 10.220.10.70/TASK: Gather Junos Facts
537030 1742792805.96971: in run() - task 54040adc-1369-87aa-6b3d-000000000004
537030 1742792805.96996: variable 'ansible_search_path' from source: unknown
537030 1742792805.97066: calling self._execute()
537030 1742792805.97174: variable 'ansible_connection' from source: group vars, precedence entry 'groups_inventory'
537030 1742792805.97194: variable 'ansible_ssh_user' from source: group vars, precedence entry 'groups_inventory'
537030 1742792805.97221: variable 'ansible_ssh_pass' from source: group vars, precedence entry 'groups_inventory'
537030 1742792805.97246: variable 'ansible_network_os' from source: group vars, precedence entry 'groups_inventory'
537030 1742792805.97267: variable 'omit' from source: magic vars
537030 1742792805.97397: variable 'omit' from source: magic vars
537030 1742792805.97458: variable 'omit' from source: magic vars
537030 1742792805.97483: variable 'omit' from source: magic vars
537030 1742792805.97511: variable 'ansible_connection' from source: group vars, precedence entry 'groups_inventory'
537030 1742792805.97537: variable 'ansible_connection' from source: group vars, precedence entry 'groups_inventory'
537030 1742792806.01265: Loading ModuleDocFragment 'ansible_collections.ansible.netcommon.plugins.doc_fragments.connection_persistent' from /usr/local/lib/python3.12/site-packages/ansible_collections/ansible/netcommon/plugins/doc_fragments/connection_persistent.py
537030 1742792806.01380: Loaded config def from plugin (connection/ansible_collections.ansible.netcommon.plugins.connection.netconf)
537030 1742792806.01430: Loading Connection 'ansible_collections.ansible.netcommon.plugins.connection.netconf' from /usr/local/lib/python3.12/site-packages/ansible_collections/ansible/netcommon/plugins/connection/netconf.py
537030 1742792806.01490: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/shell
537030 1742792806.01530: Loading ShellModule 'sh' from /usr/local/lib/python3.12/site-packages/ansible/plugins/shell/sh.py (found_in_cache=True, class_only=False)
537030 1742792806.01568: Loading ShellModule 'sh' from /usr/local/lib/python3.12/site-packages/ansible/plugins/shell/sh.py (found_in_cache=True, class_only=False)
537030 1742792806.01603: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/connection
537030 1742792806.01646: Loading Connection 'local' from /usr/local/lib/python3.12/site-packages/ansible/plugins/connection/local.py (found_in_cache=True, class_only=False)
537030 1742792806.01685: Loading ShellModule 'sh' from /usr/local/lib/python3.12/site-packages/ansible/plugins/shell/sh.py (found_in_cache=True, class_only=False)
537030 1742792806.01741: Loading ShellModule 'sh' from /usr/local/lib/python3.12/site-packages/ansible/plugins/shell/sh.py (found_in_cache=True, class_only=False)
537030 1742792806.01785: Loading Connection 'ssh' from /usr/local/lib/python3.12/site-packages/ansible/plugins/connection/ssh.py (found_in_cache=True, class_only=True)
537030 1742792806.01824: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/netconf
537030 1742792806.01869: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/netconf/__pycache__
537030 1742792806.02249: Loaded config def from plugin (netconf/ansible_collections.ansible.netcommon.plugins.netconf.default)
537030 1742792806.02306: Loading Netconf 'ansible_collections.ansible.netcommon.plugins.netconf.default' from /usr/local/lib/python3.12/site-packages/ansible_collections/ansible/netcommon/plugins/netconf/default.py (searched paths: /usr/local/lib/python3.12/site-packages/ansible/plugins/netconf:/usr/local/lib/python3.12/site-packages/ansible/plugins/netconf/__pycache__)
537030 1742792806.02366: variable 'inventory_hostname' from source: host vars for '10.220.10.70'
537030 1742792806.02419: variable 'ansible_network_os' from source: group vars, precedence entry 'groups_inventory'
537030 1742792806.02477: variable 'ansible_ssh_pass' from source: group vars, precedence entry 'groups_inventory'
537030 1742792806.02503: Set connection var ansible_shell_type to sh
537030 1742792806.02520: Set connection var ansible_shell_executable to /bin/sh
537030 1742792806.02536: Set connection var ansible_host to 10.220.10.70
537030 1742792806.02558: Set connection var ansible_pipelining to False
537030 1742792806.02575: Set connection var ansible_port to 830
537030 1742792806.02591: Set connection var ansible_timeout to 10
537030 1742792806.02606: Set connection var ansible_module_compression to ZIP_DEFLATED
537030 1742792806.02625: variable 'ansible_shell_executable' from source: unknown
537030 1742792806.02641: variable 'ansible_connection' from source: group vars, precedence entry 'groups_inventory'
537030 1742792806.02655: variable 'ansible_module_compression' from source: unknown
537030 1742792806.02671: variable 'ansible_shell_type' from source: unknown
537030 1742792806.02686: variable 'ansible_shell_executable' from source: unknown
537030 1742792806.02703: variable 'ansible_host' from source: unknown
537030 1742792806.02722: variable 'ansible_ssh_user' from source: group vars, precedence entry 'groups_inventory'
537030 1742792806.02738: variable 'ansible_ssh_pass' from source: group vars, precedence entry 'groups_inventory'
537030 1742792806.02755: variable 'ansible_port' from source: unknown
537030 1742792806.02776: variable 'ansible_pipelining' from source: unknown
537030 1742792806.02792: variable 'ansible_timeout' from source: unknown
537030 1742792806.02807: variable 'ansible_network_os' from source: group vars, precedence entry 'groups_inventory'
537030 1742792806.02822: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/action
537030 1742792806.02928: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/action/__pycache__
537030 1742792806.73192: Loading ActionModule 'ansible_collections.juniper.device.plugins.action.facts' from /root/ansible_modules_redirection_test4/ansible-junos-stdlib/ansible_collections/juniper/device/plugins/action/facts.py (searched paths: /usr/local/lib/python3.12/site-packages/ansible/plugins/action:/usr/local/lib/python3.12/site-packages/ansible/plugins/action/__pycache__) (found_in_cache=True, class_only=False)
537030 1742792806.73271: variable 'omit' from source: magic vars
537030 1742792806.73291: starting attempt loop
537030 1742792806.73308: running the handler
537030 1742792806.73333: variable 'ansible_host' from source: unknown
537030 1742792806.73348: variable 'ansible_ssh_user' from source: group vars, precedence entry 'groups_inventory'
537030 1742792806.73363: variable 'ansible_ssh_pass' from source: group vars, precedence entry 'groups_inventory'
537030 1742792806.73382: variable 'ansible_port' from source: unknown
537030 1742792806.73399: variable 'ansible_timeout' from source: unknown
537030 1742792806.73416: variable 'inventory_hostname' from source: host vars for '10.220.10.70'
537030 1742792806.73431: _low_level_execute_command(): starting
537030 1742792806.73455: _low_level_execute_command(): executing: /bin/sh -c '( umask 77 && mkdir -p "` echo /root/.ansible/tmp/ansible-local-537030phr_ny01 `"&& mkdir "` echo /root/.ansible/tmp/ansible-local-537030phr_ny01/ansible-tmp-1742792806.733973-537034-41551714035061 `" && echo ansible-tmp-1742792806.733973-537034-41551714035061="` echo /root/.ansible/tmp/ansible-local-537030phr_ny01/ansible-tmp-1742792806.733973-537034-41551714035061 `" ) && sleep 0'
537030 1742792806.73470: in local.exec_command()
537030 1742792806.73485: opening command with Popen()
537030 1742792806.73796: done running command with Popen()
537030 1742792806.73816: getting output with communicate()
537030 1742792806.74354: done communicating
537030 1742792806.74376: done with local.exec_command()
537030 1742792806.74402: _low_level_execute_command() done: rc=0, stdout=ansible-tmp-1742792806.733973-537034-41551714035061=/root/.ansible/tmp/ansible-local-537030phr_ny01/ansible-tmp-1742792806.733973-537034-41551714035061
, stderr=
537030 1742792806.74542: variable 'ansible_module_compression' from source: unknown
537030 1742792806.74600: ANSIBALLZ: Using lock for junos_facts
537030 1742792806.74629: ANSIBALLZ: Acquiring lock
537030 1742792806.74650: ANSIBALLZ: Lock acquired: 140719421142944
537030 1742792806.74677: ANSIBALLZ: Creating module
537030 1742792806.89318: ANSIBALLZ: Writing module into payload
537030 1742792806.89423: ANSIBALLZ: Writing module
537030 1742792806.89461: ANSIBALLZ: Renaming module
537030 1742792806.89505: ANSIBALLZ: Done creating module
537030 1742792806.89539: variable 'ansible_playbook_python' from source: magic vars
537030 1742792806.89578: transferring module to remote /root/.ansible/tmp/ansible-local-537030phr_ny01/ansible-tmp-1742792806.733973-537034-41551714035061/AnsiballZ_facts.py
537030 1742792806.89668: done transferring module to remote
537030 1742792806.89701: _low_level_execute_command(): starting
537030 1742792806.89726: _low_level_execute_command(): executing: /bin/sh -c 'chmod u+x /root/.ansible/tmp/ansible-local-537030phr_ny01/ansible-tmp-1742792806.733973-537034-41551714035061/ /root/.ansible/tmp/ansible-local-537030phr_ny01/ansible-tmp-1742792806.733973-537034-41551714035061/AnsiballZ_facts.py && sleep 0'
537030 1742792806.89753: in local.exec_command()
537030 1742792806.89778: opening command with Popen()
537030 1742792806.90107: done running command with Popen()
537030 1742792806.90143: getting output with communicate()
537030 1742792806.90459: done communicating
537030 1742792806.90492: done with local.exec_command()
537030 1742792806.90510: _low_level_execute_command() done: rc=0, stdout=, stderr=
537030 1742792806.90547: _low_level_execute_command(): starting
537030 1742792806.90574: _low_level_execute_command(): executing: /bin/sh -c '/usr/bin/python3 /root/.ansible/tmp/ansible-local-537030phr_ny01/ansible-tmp-1742792806.733973-537034-41551714035061/AnsiballZ_facts.py && sleep 0'
537030 1742792806.90603: in local.exec_command()
537030 1742792806.90623: opening command with Popen()
537030 1742792806.90951: done running command with Popen()
537030 1742792806.90971: getting output with communicate()
537030 1742792808.12050: done communicating
537030 1742792808.12109: done with local.exec_command()
537030 1742792808.12131: _low_level_execute_command() done: rc=1, stdout=, stderr=Traceback (most recent call last):
  File "/root/.ansible/tmp/ansible-local-537030phr_ny01/ansible-tmp-1742792806.733973-537034-41551714035061/AnsiballZ_facts.py", line 107, in <module>
    _ansiballz_main()
  File "/root/.ansible/tmp/ansible-local-537030phr_ny01/ansible-tmp-1742792806.733973-537034-41551714035061/AnsiballZ_facts.py", line 99, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File "/root/.ansible/tmp/ansible-local-537030phr_ny01/ansible-tmp-1742792806.733973-537034-41551714035061/AnsiballZ_facts.py", line 47, in invoke_module
    runpy.run_module(mod_name='ansible_collections.juniper.device.plugins.modules.facts', init_globals=dict(_module_fqn='ansible_collections.juniper.device.plugins.modules.facts', _modlib_path=modlib_path),
  File "<frozen runpy>", line 226, in run_module
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "/tmp/ansible_junos_facts_payload_wytzufm8/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/modules/facts.py", line 373, in <module>
  File "/tmp/ansible_junos_facts_payload_wytzufm8/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/modules/facts.py", line 317, in main
  File "/tmp/ansible_junos_facts_payload_wytzufm8/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/module_utils/juniper_junos_common.py", line 689, in __init__
  File "/tmp/ansible_junos_facts_payload_wytzufm8/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/module_utils/juniper_junos_common.py", line 762, in get_connection
  File "/tmp/ansible_junos_facts_payload_wytzufm8/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/module_utils/juniper_junos_common.py", line 803, in fail_json
  File "/tmp/ansible_junos_facts_payload_wytzufm8/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/module_utils/juniper_junos_common.py", line 1287, in close_configuration
AttributeError: 'JuniperJunosModule' object has no attribute '_pyez_conn'
537030 1742792808.12213: done with _execute_module (junos_facts, {'_connection': 'ansible.netcommon.netconf', 'user': 'xyz', 'passwd': 'xyz', 'host': '10.220.10.70', 'port': 830, 'timeout': 10, '_module_utils_path': '/root/ansible_modules_redirection_test4/ansible-junos-stdlib/ansible_collections/juniper/device/plugins/action', '_module_name': 'junos_facts', '_inventory_hostname': '10.220.10.70', '_ansible_check_mode': False, '_ansible_no_log': False, '_ansible_debug': True, '_ansible_diff': False, '_ansible_verbosity': 0, '_ansible_version': '2.18.3', '_ansible_module_name': 'junos_facts', '_ansible_syslog_facility': 'LOG_USER', '_ansible_selinux_special_fs': ['fuse', 'nfs', 'vboxsf', 'ramfs', '9p', 'vfat'], '_ansible_string_conversion_action': 'warn', '_ansible_socket': '/root/.ansible/pc/f1a3cdc1f2', '_ansible_shell_executable': '/bin/sh', '_ansible_keep_remote_files': False, '_ansible_tmpdir': '/root/.ansible/tmp/ansible-local-537030phr_ny01/ansible-tmp-1742792806.733973-537034-41551714035061/', '_ansible_remote_tmp': '~/.ansible/tmp', '_ansible_ignore_unknown_opts': False, '_ansible_target_log_info': None})
537030 1742792808.12235: _low_level_execute_command(): starting
537030 1742792808.12315: _low_level_execute_command(): executing: /bin/sh -c 'rm -f -r /root/.ansible/tmp/ansible-local-537030phr_ny01/ansible-tmp-1742792806.733973-537034-41551714035061/ > /dev/null 2>&1 && sleep 0'
537030 1742792808.12378: in local.exec_command()
537030 1742792808.12427: opening command with Popen()
537030 1742792808.12642: done running command with Popen()
537030 1742792808.12704: getting output with communicate()
537030 1742792808.13126: done communicating
537030 1742792808.13173: done with local.exec_command()
537030 1742792808.13209: _low_level_execute_command() done: rc=0, stdout=, stderr=
537030 1742792808.13254: handler run complete
537030 1742792808.13287: attempt loop complete, returning result
537030 1742792808.13310: _execute() done
537030 1742792808.13353: dumping result to json
537030 1742792808.13377: done dumping result, returning
537030 1742792808.13407: done running TaskExecutor() for 10.220.10.70/TASK: Gather Junos Facts [54040adc-1369-87aa-6b3d-000000000004]
537030 1742792808.13437: sending task result for task 54040adc-1369-87aa-6b3d-000000000004
537030 1742792808.13635: marking 10.220.10.70 as failed
537030 1742792808.13661: done sending task result for task 54040adc-1369-87aa-6b3d-000000000004
537030 1742792808.13682: marking host 10.220.10.70 failed, current state: HOST STATE: block=2, task=1, rescue=0, always=0, handlers=0, run_state=1, fail_state=0, pre_flushing_run_state=None, update_handlers=True, pending_setup=False, tasks child state? (None), rescue child state? (None), always child state? (None), did rescue? False, did start at task? False
537030 1742792808.13734: WORKER PROCESS EXITING
537030 1742792808.13759: ^ failed state is now: HOST STATE: block=2, task=1, rescue=0, always=0, handlers=0, run_state=5, fail_state=2, pre_flushing_run_state=None, update_handlers=True, pending_setup=False, tasks child state? (None), rescue child state? (None), always child state? (None), did rescue? False, did start at task? False
537030 1742792808.13788: getting the next task for host 10.220.10.70
537030 1742792808.13792: host 10.220.10.70 is done iterating, returning
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: AttributeError: 'JuniperJunosModule' object has no attribute '_pyez_conn'
fatal: [10.220.10.70]: FAILED! => {"changed": false, "module_stderr": "Traceback (most recent call last):\n  File \"/root/.ansible/tmp/ansible-local-537030phr_ny01/ansible-tmp-1742792806.733973-537034-41551714035061/AnsiballZ_facts.py\", line 107, in <module>\n    _ansiballz_main()\n  File \"/root/.ansible/tmp/ansible-local-537030phr_ny01/ansible-tmp-1742792806.733973-537034-41551714035061/AnsiballZ_facts.py\", line 99, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/root/.ansible/tmp/ansible-local-537030phr_ny01/ansible-tmp-1742792806.733973-537034-41551714035061/AnsiballZ_facts.py\", line 47, in invoke_module\n    runpy.run_module(mod_name='ansible_collections.juniper.device.plugins.modules.facts', init_globals=dict(_module_fqn='ansible_collections.juniper.device.plugins.modules.facts', _modlib_path=modlib_path),\n  File \"<frozen runpy>\", line 226, in run_module\n  File \"<frozen runpy>\", line 98, in _run_module_code\n  File \"<frozen runpy>\", line 88, in _run_code\n  File \"/tmp/ansible_junos_facts_payload_wytzufm8/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/modules/facts.py\", line 373, in <module>\n  File \"/tmp/ansible_junos_facts_payload_wytzufm8/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/modules/facts.py\", line 317, in main\n  File \"/tmp/ansible_junos_facts_payload_wytzufm8/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/module_utils/juniper_junos_common.py\", line 689, in __init__\n  File \"/tmp/ansible_junos_facts_payload_wytzufm8/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/module_utils/juniper_junos_common.py\", line 762, in get_connection\n  File \"/tmp/ansible_junos_facts_payload_wytzufm8/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/module_utils/juniper_junos_common.py\", line 803, in fail_json\n  File \"/tmp/ansible_junos_facts_payload_wytzufm8/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/module_utils/juniper_junos_common.py\", line 1287, in close_configuration\nAttributeError: 'JuniperJunosModule' object has no attribute '_pyez_conn'\n", "module_stdout": "", "msg": "MODULE FAILURE: No start of json char found\nSee stdout/stderr for the exact error", "rc": 1}
537030 1742792808.13908: no more pending results, returning what we have
537030 1742792808.13918: results queue empty
537030 1742792808.13921: checking for any_errors_fatal
537030 1742792808.13924: done checking for any_errors_fatal
537030 1742792808.13926: checking for max_fail_percentage
537030 1742792808.13931: done checking for max_fail_percentage
537030 1742792808.13935: checking to see if all hosts have failed and the running result is not ok
537030 1742792808.13938: done checking to see if all hosts have failed
537030 1742792808.13941: getting the remaining hosts for this loop
537030 1742792808.13945: done getting the remaining hosts for this loop
537030 1742792808.13948: getting the next task for host 10.220.10.70
537030 1742792808.13951: host 10.220.10.70 is done iterating, returning
537030 1742792808.13954: done queuing things up, now waiting for results queue to drain
537030 1742792808.13956: results queue empty
537030 1742792808.13959: checking for any_errors_fatal
537030 1742792808.13961: done checking for any_errors_fatal
537030 1742792808.13963: checking for max_fail_percentage
537030 1742792808.13966: done checking for max_fail_percentage
537030 1742792808.13968: checking to see if all hosts have failed and the running result is not ok
537030 1742792808.13970: done checking to see if all hosts have failed
537030 1742792808.13973: getting the next task for host 10.220.10.70
537030 1742792808.13976: host 10.220.10.70 is done iterating, returning

PLAY RECAP **************************************************************************************************************************************************
10.220.10.70               : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0  
